plugins/authenticator/src/u2f_crate/protocol.rs

```diff
@@ -152,7 +152,7 @@ impl U2f {
         sign_resp: SignResponse,
         counter: u32,
     ) -> Result<u32> {
-        if expiration(challenge.timestamp) > Duration::seconds(300) {
+        if expiration(challenge.timestamp) > Duration::try_seconds(300).unwrap() {
```

Comment:
```suggestion
        // Safe to unwrap since 300 is within the constraints of Duration::try_seconds
        if expiration(challenge.timestamp) > Duration::try_seconds(300).unwrap() {
```